### DISCUSSION TITLE
Missed a lot of torrents - problem with apostrophes

### DISCUSSION THREAD

**Discussion by 136785git** [cross-seed/cross-seed#438] [Q&A] [answered]
I'm new to Cross-Seed, just tried it out for the first time to seed my BHD downloads on BLU.

I did two scans, one as "safe", one as "risky", both times with includeEpisodes and includeNonVideos as true. All other settings in config were left as default.

But my scans seemed to pull up a disappointingly small number of matches (only about 30% of all the torrents it scanned), so I started manually searching BLU, and quickly found many torrents that hadn't been matched by Cross-Seed. I've manually added these to my client - they're identical in size and name, and they're verifying perfectly.

And it doesn't seem to be a problem with Jackett, as I can search for and find the aforementioned torrents in Jackett's interface.

Am I doing something wrong? Any advice would be greatly appreciated!

**Comment by bakerboy448** [comment]
Verbose logs will tell you why they were rejected

**Reply by 136785git** [reply]
Thanks for the advice. I've checked the logs - some torrents are searched for, and the logs record cross-seed comparing them to torrents on BLU, and says that they don't match. This seems to be correct, as when I manually search BLU, there is indeed no correct match. However, for the torrents that cross-seed is incorrectly missing, there is nothing recorded in the logs about them. The logs indicate that cross-seed searched for the torrent, and then 10 seconds later, moved on to the next one, there is nothing else recorded there.

**Reply by 136785git** [reply]
The missed torrents have identical file names, identical file sizes, no extra or missing files. They're completely identical, but cross-seed is missing them, with no explanation in the log.

**Reply by mmgoodnow** [reply]
run this command 

`cross-seed diff torrent1.torrent torrent2.torrent`

**Reply by 136785git** [reply]
Thanks, I tried it on a few of the torrents that cross-seed missed. All of them have the same result: "Torrents are equal".

**Reply by 136785git** [reply]
I installed cross-seed with npm, if that makes a difference. Running Node 18.16.0 on macOS Ventura.

**Reply by mmgoodnow** [reply]
Are you sure that cross-seed did a search for it?

**Reply by 136785git** [reply]
Yes, I see it in the logs. Cross-seed searches for the torrent, and 10 seconds later, moves on to the next. There are no error messages or any other information in the log.

**Reply by mmgoodnow** [reply]
That means it's probably not finding it in the search results

**Reply by 136785git** [reply]
I've had a closer look. Many of these 'missed' torrents can be explained by apostrophes.

For example, the word "Marvel's" is sometimes written as "Marvels" in the name of the torrent, but in the title of the torrent it is written as "Marvel's", thus the search fails. Or vice-versa.

Perhaps cross-seed could be made to account for this, by removing "s" and "apostrophe-s" from the end of every word? And maybe removing common words like "I'm" ?

**Reply by mmgoodnow** [reply, marked as answer]
I think the position I'm gonna take here is that it's on the tracker to have better search. Cross-seed cannot accommodate all of the search quirks of every tracker

**Reply by bakerboy448** [reply]
Cross seed should handle and support raw-search if it doesn't already.

jackett and prowlarr indicate which sites support raw search (I.e. send without apostrophe stripped) and which don't (I.e. newznabify the search string)

**Reply by removinggoose** [reply]
+1
